### ISSUE TITLE
Get latest news from python.org

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [pyhoneybot/honeybot#25]
On the landing page at python.org, fetch the latest news

```.pynews```

returns frist 5 links of news section

```
https:\\...
https:\\...
https:\\...
https:\\...
https:\\...
```

**Comment by SDeans0** [comment]
Hi, I've forked the repository and drafted this using requests_html. I'm having some trouble on actually running the bot though (mentioned on discord). I'm holding off submitting the PR until I can test it properly.

**Comment by Abdur-rahmaanJ** [comment]
closed by PR #29
